scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -12,7 +12,7 @@ workflow GvsImportGenomes {
 
     File interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
     Int? load_data_preemptible_override
-    File? load_data_gatk_override = "gs://broad-dsp-spec-ops/scratch/bigquery-jointcalling/jars/ah_var_store_20220406/gatk-package-4.2.0.0-480-gb62026a-SNAPSHOT-local.jar"
+    File? load_data_gatk_override = "gs://broad-dsp-spec-ops/scratch/bigquery-jointcalling/jars/gg_VS-360_QuietLog4J_20220412/gatk-package-4.2.0.0-489-g729c123-SNAPSHOT-local.jar"
```

Comment:
Is this meant to be permanent? My use case may be different but I was just specifying this as the optional GATK override parameter when running the workflow.

Reply:
No, this shouldn't be permanent. Really the change (if done as above) should be in the officially released gatk gar and rebased in to ah_var_store, or changed directly in ah_var_store.